frontend/device/pocketbook/device.lua

```diff
@@ -21,6 +21,10 @@ local ext_path = "/mnt/ext1/system/config/extensions.cfg"
 local app_name = "koreader.app"
 
 local PocketBook = Generic:new{
+    canUseCBB = no,
+    hasColorScreen = yes,
```

Comment:
You probably meant `= no` (generic PB has no color screen), to override it with `yes` for the color devices.
Same for hasCFAScreen = no just below.

Reply:
Temporary brute force to make CFA masking shown on all, even B&W devices (ie mine).